ApiCalls.js

```diff
@@ -24,4 +24,22 @@ export const addFlyToAPI = async (flyData) => {
   } catch (error) {
     console.error(error);
   }
-};
\ No newline at end of file
+};
+
+export const deleteFly = async (flyId) => {
+    try {
+      const response = await fetch(
+        `https://my-fly-box-api.herokuapp.com/api/v1/flies/${flyId}`,
+        {
+          method: "DELETE",
+          headers: {
+            "Content-Type": "application/json;charset=utf-8",
+          },
+        }
+      );
+      const deletionResponse = await response.json();
+      console.log(deletionResponse);
+    } catch (error) {
+      console.error(error);
+    }
```

Comment:
Is this the error handling before it hits the back end? We only have error for creating and display a specific one by id.

Reply:
No this would be if there was an error trying to delete a fly - for example, if the fly ID that the user is trying to delete doesn't exist it would be "Couldn't find Fly with 'id'=____"